sysadmin/phpmyadmin/config.inc.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/* vim: set expandtab sw=4 ts=4 sts=4: */
+/**
+ * phpMyAdmin sample configuration, you can use it as base for
+ * manual configuration. For easier setup you can use setup/
+ *
+ * All directives are explained in documentation in the doc/ folder
+ * or at <https://docs.phpmyadmin.net/>.
+ *
+ * @package PhpMyAdmin
+ */
+
+/**
+ * This is needed for cookie based authentication to encrypt password in
+ * cookie
+ */
+$cfg['blowfish_secret'] = 'ThisISPhpMyADMINforAlltheflightdeals\'SNOTREALLY'; /* YOU MUST FILL IN THIS FOR COOKIE AUTH! */
```

Comment:
Not sure, but this looks like it should be read in via an environment variable.

Reply:
I added a comment
```php7
/**
 * This is needed for cookie based authentication to encrypt password in
 * cookie.
 * 
 * Note that it's OK for this to be embedded in this doc in clear-text:
 * it's used to encrypt cookies at rest. Using the secret requires someone to
 * have also accessed the database and a user's cookies. Per phpMyAdmin docs,
 * it's fine to just fill this in in the file rather than using an environment
 * variable:
 * https://docs.phpmyadmin.net/en/latest/setup.html#manually-creating-the-file
 */
```